proto/key_provisioning/key_provisioning.proto

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.key_provisioning.v1;
+
+import "oak_crypto/proto/v1/crypto.proto";
+import "proto/attestation/evidence.proto";
+import "proto/attestation/endorsement.proto";
+
+message GetProvisionSecretsRequest {
+  oak.attestation.v1.Evidence evidence = 1;
+  oak.attestation.v1.Endorsements endorsements = 2;
+}
+
+message GetProvisionSecretsResponse {
+  // Encryption private key that was encrypted with HPKE using the encryption public key provided
+  // in the endorsed evidence.
+  oak.crypto.v1.EncryptedRequest encrypted_encryption_key = 1;
+}
+
+// Defines the Key Provisioning Service that distributes keys between enclaves.
+service KeyProvisioning {
+  // Request provision secrets from for other enclaves as part of Key Provisioning.
```

Comment:
I commented on this on the various other CLs, I am not sure "provision secret" is grammatically correct. I used "provision" as verb when discussing on the docs (i.e. `rpc ProvisionSecrets`), but not as adjective. But anyways, I think yesterday we agreed to call that group key, so probably we should use that?

Reply:
Renamed to `GetGroupKeys` and `SendGroupKeys`